src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+/* #region header */
+
+ul {
+  list-style: none;
+}
+
+.header {
+  box-sizing: content-box;
```

Comment:
The `box-sizing: content-box;` is not necessary here since you've already defined `box-sizing: border-box;` globally on line 12. Unless you have a specific reason to change the box-sizing for `.header` only, it's better to stick with the global setting for consistency.